tests/utils/test_utils.py

```diff
@@ -156,3 +157,92 @@ def test_rotate_cell(self):
 
         newdoc.validate(recursive=True)
         write_neuroml2_file(newdoc, "tests/utils/test_rotation.net.nml", validate=True)
+
+    def test_translate_cell_to_coords(self):
+            """Test translate_cell_to_coords"""
+            acell = neuroml.utils.component_factory(
+                "Cell", id="test_cell", validate=False
+            )  # type: neuroml.Cell
+
+            soma = acell.add_segment(
+                prox=[10, 10, 10, 15],
+                dist=[10, 10, 10, 15],
+                seg_id=0,
+                use_convention=False,
+                reorder_segment_groups=False,
+                optimise_segment_groups=False,
+            )
+
+            acell.add_segment(
+                prox=[10, 10, 10, 12],
+                dist=[110, 10, 10, 12],
+                seg_id=1,
+                use_convention=False,
+                reorder_segment_groups=False,
+                optimise_segment_groups=False,
+                parent=soma,
+            )
+
+            acell.add_segment(
+                prox=[10, 10, 10, 7],
+                dist=[10, 160, 10, 7],
+                seg_id=2,
+                use_convention=False,
+                reorder_segment_groups=False,
+                optimise_segment_groups=False,
+                parent=soma,
+            )
+
+            acell.add_segment(
+                prox=[10, 10, 10, 4],
+                dist=[10, 10, 210, 4],
+                seg_id=3,
+                use_convention=False,
+                reorder_segment_groups=False,
+                optimise_segment_groups=False,
+                parent=soma,
+            )
+
+            print(f'cell before: {acell}')
+
+            translated_cell = translate_cell_to_coords(acell, False, [30,30,30])
```

Comment:
how about an assertion to test that the soma is now at `[30, 30, 30]`, and that a few of the segments have moved by the correct amount? (I couldn't figure out an easy way to test out rotation, but we should be able to have a few basic checks for translation?)